code/game/machinery/recycler.dm

```diff
@@ -92,10 +92,10 @@
 /obj/machinery/recycler/Bump(atom/movable/AM)

```

Comment:
This whole proc can probably be deleted, it's nto doing anything it's parent procs arent